### PR TITLE
Update Nest component configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
**Description:**
Update style of Nest component documentation to follow new configuration variables description.
Related to #6385.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Strange I do not see an error message in the deploy preview 🤔

File: source/_components/sensor.nest.markdown [hunk 1]
```diff
@@ -34,9 +34,12 @@ nest:
...
 By default all sensors for your available Nest devices will be monitored. Leave `monitored_conditions` blank to disable all sensors for the [Nest component](/components/nest/).
 
-Configuration variables:
-
-- **monitored_conditions** array (*Optional*): States to monitor.
+{% configuration %}
+monitored_conditions:
+  description: States to monitor.
+  required: false
+  type: list
```
Comment (frenck) [inline source/_components/sensor.nest.markdown:41]: Please add the list of monitored conditions using the `keys:` option.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@klaasnicolaas The build will probably pick up again after the next commit.

File: source/_components/sensor.nest.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/sensor.nest.markdown:41]: The list below? How do I do that with all the sub-items that are there?

**Comment by klaasnicolaas** [comment]
Can it still be merged? the adjustments were ultimately not necessary 😉

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @klaasnicolaas! 👍 

Happy Hacktoberfest 🎉

**Pull request merged**